Application/src/Utils/Text.js

```diff
@@ -84,3 +32,41 @@ export async function getText(image) {
 
   return obj;
 }
+
+const findText = async (image, grade) => {
+  // regex that matches 2+ alphabetic characters in string
+  const regex = /([a-zA-Z]{2,})/g;
```

Comment:
Do not comment bad code, rewrite it...
For example, calling an regex object "regex" is like naming a child, "Child". Consider renaming the regex object to "TwoAlphaCharMatchedRegexExpression" or something that tells me as a fellow contributor about the regex object that I learn by reading the variable name without needing a comment explaining it.

Reply:
I named it 'regex' because it is literally just regex. Personally, I find more concise variable names, with a helpful comment, make the code easier to read. I might remove the regex code altogether since I don't need it now that spell checking can't really be done.

Reply:
It doesn't help to have a vague concise variable because, if the variable is referenced later then you will have no understanding of what the variable does. The comment explains what the variable is for and where the variable was declared could be hundreds of lines of code further up the page or it was declared in another file.